react/src/pages/EndpointDetailPage.tsx

```diff
@@ -598,7 +596,7 @@ const EndpointDetailPage: React.FC<EndpointDetailPageProps> = () => {
               title: 'Created at',
               dataIndex: 'created_at',
               render: (text, row) => (
-                <span>{dayjs(row.created_at).format('ll LT')}</span>
+                <span>{dayjs(row?.created_at).format('ll LT')}</span>
```

Comment:
Row can not be `undefined`. Is there any reason to add `?.`?

Reply:
I thought `created_at` should be null or undefined. I'll check query data type and apply optional chaining if have null or undefined value.

Reply:
Arrays for undefined or null are deleted when using filterNonNullItems, so rows can't be undefined. I'll fix this.